docs/cookbook/sylius-cloud/introduction-to-sylius-cloud.rst

```diff
@@ -0,0 +1,124 @@
+- \- Infrastructure as a code - all you need to do is to write your application and a few Yaml files to create your environment. So simple and so powerful.
+- \- Multi-services and multi-apps - it doesn't matter your application is monolith- or microservice-based architecture. It doesn't matter how much different external services you need (MySQL, PostgreSQL, ElasticSearch, Redis etc.) - it just works good in all cases.
+
+Sylius Cloud configuration in Sylius
+------------------------------------
+
+All the infrastructure configuration files are placed in `Sylius-Standard <https://github.com/Sylius/Sylius-Standard>`_ project. It means you'll get it when running the command:
+
+.. code-block:: bash
+
+    composer create-project sylius/sylius-standard acme
+
+The paths you should be interested in are:
+
+- \- `.platform.app.yaml` - This file is used to define the configuration of the application itself. It specifies how the application should be built, what runtime to use, which commands to run on deployment, etc. It also includes information such as the type of application, language runtime, build and deploy hooks, and more.
+- \- `.env.platform.dist` - The environment file automatically used by the application during the build process.
+- \- `.platform/routes.yaml` - This file defines the routes and how traffic is routed to the application. It specifies the incoming URLs and how they should be directed to the appropriate services or applications. Routes can be configured to handle different paths, domains, or other conditions.
+- \- `.platform/services.yaml` - This file is used to define additional services or dependencies required by the application. It allows defining various types of services such as databases, caches, search engines, etc., along with their configuration. Services defined in this file are automatically provisioned and integrated with the application environment.
+
+Creating a new project and environment
+--------------------------------------
+
+Every new environment needs to be created in the Sylius Cloud Console or by using the CLI.
+But first before creating an environment you need to create the project. When you're creating a new project from the Console,
+to use the Sylius Cloud potential, please choose the "Create from scratch" option, as selected on the screenshot below:
+
+.. image:: ../../_images/cookbook/sylius-cloud/create-from-scratch.png
+    :align: center
+    :scale: 50%
+
+|
+
+During creating a new project you'll automatically create a new environment, which is connected to your git branch (please read the information on screenshot below):
+
+.. image:: ../../_images/cookbook/sylius-cloud/create-from-scratch-default-environment.png
+    :align: center
+    :scale: 50%
+
+|
+
+When you've created the project, you're ready to create new environments. Please note that your project already have the first (production) environment just after project is being created.
```

Comment:
I'm wondering if We would like to emphasize further that environments are essentially open branches that could be used in the name of testing, development, new features like you said in the line 89